### PR TITLE
Fix spelling/typos

### PR DESCRIPTION
**Pull request by SuperQ**
Fix spelling/typos reported by codespell/misspell.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func (c *GlobalConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
-// isZero returns true iff the global config is the zero value.
+// isZero returns true if the global config is the zero value.
```
Comment (brian-brazil) [inline config/config.go]: this was correct

File: notifier/notifier.go [hunk 2]
```diff
@@ -88,7 +88,7 @@ func (a *Alert) String() string {
 	return s + "[active]"
 }
 
-// Resolved returns true iff the activity interval ended in the past.
+// Resolved returns true if the activity interval ended in the past.
```
Comment (brian-brazil) [inline notifier/notifier.go]: ditto

Comment (SuperQ) [reply notifier/notifier.go]: Fixed

File: config/config.go (hunk 1 above)
Comment (SuperQ) [reply config/config.go]: Fixed.

**Comment by brian-brazil** [comment]
:+1:

**Comment by mjtrangoni** [comment]
@SuperQ PTAL, related to US vs UK spelling, this could be fixed too, 
````$ find . -type f | xargs misspell -locale US | grep -v vendor````

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**